src/frontend/components/horizontalCard/horizontal-card.js

```diff
@@ -0,0 +1,65 @@
+import { useCart } from "../../contexts/cart-context";
+import "./horizontal-card.css";
+const laptopImage = require('../../assets/laptop.png');
+
+
+const HorizontalCard = (product)=>{
+  
+    const {removeFromCart}= useCart()
+    const incrementQuantity=(product)=>{
+        // {cart.map((item)=>item._id===product._id) ? {...item, quantity:product.quantity+1} :item
```

Comment:
remove commented code

Reply:
will do it next PR, this code is going to use for incrementing quantity